tests/test_docs/test_skill_guide/test_skill_guide.py

```diff
@@ -62,13 +62,11 @@ def setup_class(cls):
         cls.doc_path = os.path.join(ROOT_DIR, MD_FILE)
         cls.code_blocks = extract_code_blocks(filepath=cls.doc_path, filter_="python")
 
-    def test_read_md_file(self):
-        """Teat that the md file is not empty."""
-        assert self.code_blocks != [], "File must not be empty."
-
     @pytest.mark.flaky(reruns=MAX_FLAKY_RERUNS_INTEGRATION)
     def test_update_skill_and_run(self):
         """Test that the resource folder contains scaffold handlers.py module."""
+        assert self.code_blocks != [], "File must not be empty."
```

Comment:
It makes sense to call it here as the method further in relies on the code_blocks being non-empty.